client/my-sites/domains/domain-management/list/domain-item.jsx

```diff
@@ -146,12 +146,9 @@ class DomainItem extends PureComponent {
 			return;
 		}
 
-		const autoRenewLabel = translate( 'Auto-renew: %(autoRenewValue)s', {
-			args: {
-				autoRenewValue: domainDetails?.isAutoRenewing ? 'on' : 'off',
-			},
-			comment: 'Auto-renew a domain registration. %(autoRenewValue)s can be "on" or "off"',
-		} );
+		const autoRenewLabel = domainDetails?.isAutoRenewing
+			? translate( 'Auto-renew: on' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **25** times:
`translate( 'Auto-renew (on)' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/auto-renew-not-translated)